locksmith/src/controllers/metadataController.ts

```diff
@@ -4,14 +4,56 @@ import { Response } from 'express-serve-static-core' // eslint-disable-line no-u
 import Normalizer from '../utils/normalizer'
 import { SignedRequest } from '../types' // eslint-disable-line no-unused-vars, import/no-unresolved
 import LockData from '../utils/lockData'
-
 import { addMetadata } from '../operations/userMetadataOperations'
+import { KeyHoldersByLock } from '../graphql/datasource/keyholdersByLock'
 
 const env = process.env.NODE_ENV || 'development'
 const config = require('../../config/config')[env]
 const metadataOperations = require('../operations/metadataOperations')
+const lockOperations = require('../operations/lockOperations')
 
 namespace MetadataController {
+  const evaluateLockOwnership = async (
+    lockAddress: string,
+    signeeAddress: string
+  ) => {
+    const lockData = new LockData(config.web3ProviderHost)
+
+    return (
+      Normalizer.ethereumAddress(signeeAddress) ===
+      Normalizer.ethereumAddress(await lockData.owner(lockAddress))
+    )
+  }
+
+  const expiredSignature = (
```

Comment:
Wouldn;t this make sense as a helper/util?

Reply:
yeah, thats a good call will move elsewhere